src/opm/output/eclipse/LoadRestart.cpp

```diff
@@ -999,33 +982,18 @@ namespace {
             return;
         }
 
-        const auto& conn0 = well.getConnections();
-        const auto conns = Opm::WellConnections(conn0, grid);
-        const auto ijk_to_res = ijk_to_resID(wellID, nConn, wellData);
-
-        auto linConnID = std::size_t{0};
-        for (const auto& conn : conns) {
-            if (++linConnID > nConn) { continue; }
-
-            auto& xc = xw.connections[linConnID - 1];
-
-            const auto ijk =
-                std::make_tuple(conn.getI(), conn.getJ(), conn.getK());
-
-            auto resPos = ijk_to_res.find(ijk);
-
-            if (resPos != std::end(ijk_to_res)) {
-                const auto connID = resPos->second;
-                const auto xcon   = wellData.xcon(wellID, connID);
-
-                restoreConnRates(xcon, usys, oil, gas, wat, xc);
-
-                xc.index = grid.getGlobalIndex(std::get<0>(ijk),
-                                               std::get<1>(ijk),
-                                               std::get<2>(ijk));
+        for (std::size_t rstConnID = 0; rstConnID < nConn; rstConnID++) {
+            const auto icon = wellData.icon(wellID, rstConnID);
+            const auto i = icon[VI::IConn::index::CellI] - 1;
+            const auto j = icon[VI::IConn::index::CellJ] - 1;
+            const auto k = icon[VI::IConn::index::CellK] - 1;
+            auto * xc = xw.find_connection(grid.getGlobalIndex(i,j,k));
+            if (!xc)
+                continue;
```

Comment:
Braces please.

Reply:
No

Reply:
>> Braces please.
>
> No

Okay, then please be advised that I'm going to reinsert the braces very soon.

Reply:
![image](https://user-images.githubusercontent.com/1321665/77413770-2cbdb500-6dc0-11ea-8dad-a88611661b3f.png)

I'm the guy in the blue jacket :-)

> Okay, then please be advised that I'm going to reinsert the braces very soon.

In that case I will not remove them again.

Reply:
What happened to keeping the existing style of the file? (Not having looked at the file in question, so I do not know if that would lead to bracers or not.)